public-api/src/integration/java/org/pmiops/workbench/google/BuildIapRequest.java

```diff
@@ -0,0 +1,55 @@
+package org.pmiops.workbench.google;
+
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdTokenCredentials;
+import com.google.auth.oauth2.IdTokenProvider;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ Class that can be used to get OIDC token for a service account.
+ https://cloud.google.com/iap/docs/authentication-howto#iap_make_request-java
+ */
+public class BuildIapRequest {
+    private static final String IAM_SCOPE = "https://www.googleapis.com/auth/iam";
+
+    public BuildIapRequest() throws IOException {
```

Comment:
opt: Probably just use a private constructor in this case, since it's not meant to be instantiated

Reply:
Removed constructor. Thought it will. use java's default constructor.